build.csx

```diff
@@ -16,7 +16,7 @@ var resharperCltUrl = new Uri("https://download.jetbrains.com/resharper/JetBrain
 var resharperCltPath = $"{Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData)}/.resharper/{resharperCltUrl.Segments.Last()}";
 var inspectCodePath = $"./.resharper/{Path.GetFileNameWithoutExtension(resharperCltUrl.Segments.Last())}/inspectcode.exe";
 var nuget = "src/.nuget/v4.0.0/NuGet.exe";
-var msBuild = $"{Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86)}/MSBuild/14.0/Bin/msbuild.exe";
```

Comment:
MSBuild 15 has moved inside the Visual Studio installation, which means we can't guarantee this path is valid any more.

If we don't want to assume Professional, we need to pull in https://github.com/Microsoft/vswhere to actually figure it out at run time.